whipper/src/main/java/org/whipper/ExpectedResultHolder.java

```diff
@@ -42,6 +45,23 @@ public void buildResult(File xmlFilePath, Query q) throws IOException{
         XmlHelper.loadResult(xmlFilePath, this, q);
     }
 
+    /**
+     * Used for building a fake result in case when a file with the expected result is missing
+     */
+    public void buildFakeResultFromMissingFile(String error, Query q) throws IOException {
+        clear();
+        String xml = "<?xml version=\"1.0\"?>\n" +
```

Comment:
I do not like this.
1. Query is missing - this is required element.
2. If we change XSD schema, we need to update it here too.
3. We have Java classes for this.
4. If you take a look at "XmlHelper.loadResult" - if there is no result, then only method called on ERH is erh.setOriginalResult(qr) so there is no need to write new files.

I believe, all you need is to clear ERH, set Original result to no-result (could be constant somewhere) and add one error.

Reply:
I now set the original result to null